integration-libs/cpq-quote/cpq-quote-discount/components/cpq-quote-discount-tbody/cpq-quote.component.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, Optional, OnDestroy, OnInit, Inject } from '@angular/core';
+import { CartItemContext, OrderEntry } from '@spartacus/cart/base/root';
+import { CpqDiscounts } from '../../../root/model';
+
+import { EMPTY, Observable, Subscription } from 'rxjs';
+import { CpqQuoteService } from '../../cpq-qute.service';
+interface ExtendedOrderEntry extends OrderEntry {
+  cpqDiscounts?: CpqDiscounts[];
+}
+
+@Component({
+  selector: 'cx-cpq-quote',
+  templateUrl: './cpq-quote.component.html',
+  styleUrls: ['./cpq-quote.component.scss'],
```

Comment:
This line should be removed after `cpq-quote.component.scss` file has been moved to `styles` folder